### ISSUE TITLE
No visible interface for startAnimation function when using Obj-C

### ISSUE THREAD

**Issue by Gringox** [PiXeL16/RevealingSplashView#33]
Hello, I am trying to import this module as a part of my iOS app build with Objective-C. I found that the interface for the startAnimation function does not get recognized anywhere. Any help?

PS: I am using CocoaPods.

**Comment by PiXeL16** [comment]
Hey @Gringox, will take a look when I have time. I have not really use this in `Obj-C` before, its probably missing some `@objc` attributes.
`PR`s are very welcome as well!

**Comment by Gringox** [comment]
Thank you for your quick answer @PiXeL16.

I did manage to narrow it down to this [issue](https://bugs.swift.org/browse/SR-492) and I did manage to solve it by changing the SplashAnimatable extension (which is a protocol extension) to a RevealingSplashView extension (which is a class extension) and that way the methods get the visibility.

So basically I changed this line in Animations.swift from:

`public extension SplashAnimatable where Self: UIView {
`

To:

`public extension RevealingSplashView {
`

I did not make a pull request of this because I think it may mess up some of the order you originally structure the project.

So I want to ask you if you find this change ok?
